src/content/en/2020/markup.md

```diff
@@ -57,7 +57,7 @@ In this section, we're covering the higher-level aspects of HTML like document t
       <tr>
         <th>Doctype</th>
         <th>Pages</th>
-        <th>Percentage</th>
+        <th>Pages (%)</th>
```

Comment:
I’ve commented on the general issue of changing the headers/labels in #467; this _seems_ fine, however.

Reply:
Yeah tried to standardise on this when I could since it was used in some places but not others.